deploy/role.yaml

```diff
@@ -0,0 +1,67 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: antrea-operator
+rules:
+- apiGroups:
+  - ""
+  resources:
```

Comment:
How about we only add what we need for now, and add more later?

Reply:
Let's add rules when need them.